src/views/Overview.vue

```diff
@@ -29,8 +29,11 @@ import RecentTransactions from "@/components/RecentTransactions"
 import Statistics from "@/components/Statistics"
 import SummaryHero from "@/components/SummaryHero"
 
+import superagent from 'superagent'
 import { fetchBlocks, fetchStakeStats, fetchTransactions } from '../utils/api'
 
+const mapRefreshInterval = 60 * 1000
```

Comment:
Any reason for this not being in the data like `pollInterval`?

Reply:
In a previous component that I did this is where Anny asked me to put the polling/refresh intervals. For consistency, I guess I could move the poll interval out of `data`, and go through Wallet and try to do that throughout. There is still a bit of a mix of approaches